### DISCUSSION TITLE
Usb drive mount

### DISCUSSION THREAD

**Discussion by talgius** [fayaaz/mixxx-pi-gen#37] [Q&A] [answered]
Hi, I tried to put a USB pen drive to my raspberry pi 4b running the latest mixxx-pi-gen, but it doesn't seem to be mounted. 
There is a command to force the mount or something I can do to automatically mount the pen drive?
Thanks, all the best.

Edit: Sorry, I solved but I cannot delete the question.

**Comment by fayaaz** [comment]
@talgius how did you solve it? I thought udevil would take care of all of that - was there a problem with your drive?

**Reply by talgius** [reply, marked as answer]
I was misled by the fact that once the USB drive was attached nothing happened. From the setting of Mixxx I manually added the folder under Media to scan the drive and solve the problem.
Thank you anyway.
